### PR TITLE
KubeCon EU 2024 Blog Post

### PR DESCRIPTION
**Pull request by svrnm**
An early draft for the semi-yearly KubeCon blog post. A few details are still missing, I'll fill them out later.

cc @open-telemetry/governance-committee

### CODE REVIEW

**Review by mtwo — APPROVED** [review]

**Review by cartermp — APPROVED** [review]
LGTM (even if it's just a draft)

**Review by alolita — APPROVED** [review]
Lgtm @svrnm :-) even in draft version. 

Are you planning to do another post for the colocated Observability Day talks?

**Comment by svrnm** [comment]
> Lgtm @svrnm :-) even in draft version.
> 
> Are you planning to do another post for the colocated Observability Day talks?

I think I will list the otel related talks in this blog post as well. I just need to find the time to copy all of them out of the agenda.

**Comment by avillela** [comment]
@svrnm re: the OTel Observatory, I'm working with @reese-lee and the rest of the OTel EUWG to plan out events/activities in the booth. @AnaMMedina21 is putting together a schedule for us, which we can publish here. We're also holding a few activities that require sign-ups, like Humans of OTel and End User feedback sessions. I'll share the links once we have those finalized.

**Comment by svrnm** [comment]
> @svrnm re: the OTel Observatory, I'm working with @reese-lee and the rest of the OTel EUWG to plan out events/activities in the booth. @AnaMMedina21 is putting together a schedule for us, which we can publish here. We're also holding a few activities that require sign-ups, like Humans of OTel and End User feedback sessions (@musingvirtual and @danielgblanco are planning the feedback sessions). I'll share the links once we have those finalized.

Thanks! I updated the section about the observagory, please take a look. We can hold back the blog post for a little bit, but maybe should publish it end of next week, early March.

**Comment by musingvirtual** [comment]
Honeycomb drafts our vendor agnostic OpenTelemetry email on Feb 29th to
send on March 1, be great to get it out by then so we can include the post!

**Comment by svrnm** [comment]
> Honeycomb drafts our vendor agnostic OpenTelemetry email on Feb 29th to send on March 1, be great to get it out by then so we can include the post!

Then we aim for Feb 28th or something:-)

File: content/en/blog/2024/kubecon-eu.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+your OpenTelemetry adoption, implementation, and usage. We will create action
+items from your comments as appropriate. Check [#otel-user-research][] in CNCF's
+Slack instance for results and action item updates to come after KubeCon EU.
+
+## OpenTelemetry Observatory
+
+In addition to our booth at the Project Pavilion, please visit the OpenTelemetry
+Observatory presented by Splunk in the Expo Hall! This will be a place for
+informal chats, meetups, and other discussions led by OpenTelemetry community
+members and maintainers. If you’d like to participate and lead a discussion or
```
Comment (avillela) [inline content/en/blog/2024/kubecon-eu.md:117]: @svrnm we'll actually be posting a schedule and sign-ups for the various sessions. We can link to these directly from the blog post.

Comment (svrnm) [reply content/en/blog/2024/kubecon-eu.md]: awesome, share the links with me when you have them. We can also add them in after publishing the blog post

**Comment by svrnm** [comment]
@open-telemetry/end-user-wg @open-telemetry/governance-committee please take another look, I was hoping to publish this by Wednesday.

File: content/en/blog/2024/kubecon-eu.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+{{% alert title="Important access note" color="danger" %}}
+
+You need an _in-person all-access_ pass for on-site access to **Observability
+Day**. For details, see [KubeCon registration][]. If you have a virtual ticket,
+you will be able to follow **Observability Day** through a live stream.
+
+{{% /alert %}}
+
+## OpenTelemetry Observatory
```
Comment (danielgblanco) [inline content/en/blog/2024/kubecon-eu.md:94]: I know we're still shaping up the feedback sessions at the observatory, but is this something we want to mention here @open-telemetry/end-user-wg ?

File: content/en/blog/2024/kubecon-eu.md (hunk 1 above)
Comment (avillela) [reply content/en/blog/2024/kubecon-eu.md]: Perfect! Hope to have somthing to you this week!

File: content/en/blog/2024/kubecon-eu.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
+---
+title: Join us for OpenTelemetry Talks and Activities at KubeCon EU 2024
+linkTitle: KubeCon NA '24
```
[the remark this answers is not in the document]
Comment (svrnm) [reply content/en/blog/2024/kubecon-eu.md:3]: oh wow, how could I miss that! Thanks!!

[the remark this answers is not in the document]
Comment (svrnm) [reply content/en/blog/2024/kubecon-eu.md:3]:
```suggestion
linkTitle: KubeCon EU '24
```

File: content/en/blog/2024/kubecon-eu.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+- **[Prometheus and OpenTelemetry: Better Together](https://sched.co/1YePz)**<br>
+  by Adriana Villela, ServiceNow Cloud Observability & Reese Lee, New Relic<br>
+  Thursday, March 21 • 16:30 - 17:05
+- **[Observable Feature Rollouts with OpenTelemetry and OpenFeature](https://sched.co/1YeSC)**<br>
+  by Daniel Dyla & Michael Beemer, Dynatrace<br> Friday, March 22 • 16:00 -
+  16:35
+
+## Co-located Events
+
+[Observability Day][] _fosters collaboration, discussion, and knowledge sharing
```
[the remark this answers is not in the document]
Comment (svrnm) [reply content/en/blog/2024/kubecon-eu.md:45]:
```suggestion
_[Observability Day][] fosters collaboration, discussion, and knowledge sharing
```

File: content/en/blog/2024/kubecon-eu.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+{{% /alert %}}
+
+## OpenTelemetry Observatory
+
+Drop by and say _"Hi!"_ at OpenTelemetry Observatory presented by Splunk in the
+Expo Hall. This will be a place for informal chats, meetups, and other
+discussions led by OpenTelemetry community members and maintainers. If you’d
+like to participate and lead a discussion or short presentation, please reach
+out to the OpenTelemetry End User Working Group to indicate your interest.
```
Comment (svrnm) [inline content/en/blog/2024/kubecon-eu.md:100]:
```suggestion
out to the [OpenTelemetry End User Working Group](https://cloud-native.slack.com/archives/C01RT3MSWGZ) to indicate your interest.
```

**Review by avillela — COMMENTED** [review]
Left some in-line comments @svrnm

File: content/en/blog/2024/kubecon-eu.md (hunk 2 above)
Comment (avillela) [reply content/en/blog/2024/kubecon-eu.md:94]: @danielgblanco most definitely - I added some suggestions around that

**Review by Kielek — APPROVED** [review]

**Review by tedsuo — APPROVED** [review]

File: content/en/blog/2024/kubecon-eu.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+title: Join us for OpenTelemetry Talks and Activities at KubeCon EU 2024
+linkTitle: KubeCon EU '24
+date: 2024-02-28
+# prettier-ignore
+cSpell:ignore: Aiven Alexandre Anusha Arbiv Beemer Benedikt Blanco Bongartz Chekuri Coralogix Cosmonic Dyrmishi Jiekun Joonas Kanal Kolachala Kowall Machado Magno Marcin Matej Mirabella Narapureddy Nenashev Oleg Oluwalolope Outshift Pismo Purvi Quwan Reddy Ridwan Rollouts Ryanair Skyscanner Sodkiewicz Soluções Srikanth Tecnológicas Yosef
+author: '[Severin Neumann](https://github.com/svrnm) (Cisco)'
+---
+
+The OpenTelemetry project maintainers, members of the governance committee, and
+technical committee are thrilled to be at [KubeCon EU][] in Paris from March
```
Comment (jpkrohling) [inline content/en/blog/2024/kubecon-eu.md:11]:
```suggestion
technical committee are thrilled to be at [KubeCon + CloudNativeCon Europe][] in Paris from March
```

Comment (jpkrohling) [reply content/en/blog/2024/kubecon-eu.md:11]: "KubeCon + CloudNativeCon Europe" is the actual name of the conference :-)

Comment (jpkrohling) [reply content/en/blog/2024/kubecon-eu.md:11]: Should we also include Observability Day, co-located with KubeCon?

**Review by jpkrohling — APPROVED** [review]
Small nit on the conference name, LGTM otherwise.

File: content/en/blog/2024/kubecon-eu.md (hunk 6 above)
Comment (svrnm) [reply content/en/blog/2024/kubecon-eu.md:11]: Good call out! Yes, we can have Observability Day as well

Comment (svrnm) [reply content/en/blog/2024/kubecon-eu.md:11]:
```suggestion
technical committee are thrilled to be at [KubeCon + CloudNativeCon Europe][] and at the co-located [Observability Day](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/observability-day/) in Paris from March
```

File: content/en/blog/2024/kubecon-eu.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
+---
+title: Join us for OpenTelemetry Talks and Activities at KubeCon EU 2024
```
Comment (svrnm) [inline content/en/blog/2024/kubecon-eu.md:2]:
```suggestion
title: Join us for OpenTelemetry Talks and Activities at KubeCon + CloudNativeCon Europe 2024
```

**Comment by svrnm** [comment]
https://opentelemetry.io/blog/2024/kubecon-eu/

**Pull request merged**